lib/rzo/trollop.rb

```diff
@@ -0,0 +1,926 @@
+# lib/trollop.rb -- trollop command-line processing library
```

Comment:
It's based on my experience working with Trollop in Puppet core.  Trollop is designed to be vendored, and it avoids issues in the situation two different command line utilities depend on different versions of Trollop.